Signum.Entities.Extensions/Authorization/AuthMessages.cs

```diff
@@ -79,6 +79,13 @@ public enum AuthMessage
         NotAuthorizedTo0The1WithId2,
         [Description("Not authorized to Retrieve '{0}'")]
         NotAuthorizedToRetrieve0,
+        PasswordsAreDifferent,

```

Comment:
This messages where not removed from `AuthMessage`, but moved to `LoginAuthMessage`. Probably you need them for the merge errors after the move from `DoublePassword`. This merge problems are already fixed in `master`.

Reply:
ok

Reply:
You need to remove all this new keys